vehicle/tests/golden/compile/acasXu/VNNLib.queries/property8-query3.txt.golden

```diff
@@ -0,0 +1,31 @@
+; WARNING: This file was generated automatically by Vehicle
+; and should not be modified manually!
+; Metadata:
+;  - VNNLib query format version: unknown
+;  - Vehicle version: 0.16.1+dev
+
+(assert (<= (- () acasXu@0Y_1) 0.0))
```

Comment:
Minor but it would probably be nicer if we used the seperator character `@` between the index and the i/o status as well, e.g. `acasXu@0@Y_1`?